--- origsrc/gimp-texturize/src/offset.c	2012-08-10 14:21:44.315491800 -0500
+++ src/gimp-texturize/src/offset.c	2012-08-10 14:24:58.424012400 -0500
@@ -63,7 +63,7 @@ offset_optimal (gint    *resultat,
                 gint     channels, guchar **rempli,
                 gboolean tileable) {
   gint x_i, y_i;
-  float best_difference = HUGE, tmp_difference;
+  float best_difference = HUGE_VAL, tmp_difference;
 
   for (x_i = x_patch_posn_min; x_i < x_patch_posn_max; x_i++) {
     for (y_i = y_patch_posn_min; y_i < y_patch_posn_max; y_i++) {
